packages/peregrine/lib/store/actions/cart/asyncActions.js

```diff
@@ -25,7 +25,9 @@ export const createCart = payload =>
 
         try {
             // errors can come from graphql and are not thrown
-            const { data, errors } = await fetchCartId();
+            const { data, errors } = await fetchCartId({
+                fetchPolicy: 'no-cache'
```

Comment:
We were storing the cart id result under ROOT_MUTATION. We don't need to/shouldn't store this.